ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -150,6 +153,7 @@ public ForkChoice(
         new TickProcessor(spec, recentChainData),
         transitionBlockValidator,
         true,
+        true,
```

Comment:
assuming this is just a testing constructor if we're hard coding here?

Reply:
yeah this is just old deprecated constructor used in some old tests